### ISSUE TITLE
Add Self Made Projects under respective domain - NO NEED TO GET ASSIGNED

### ISSUE THREAD

**Issue by kaal-coder** [kaal-coder/hacktoberfest2023#151]
Add your self coded codes in the Respective Domain Folder.
There is no need to get assigned for this issue.
Make sure you give a Star to the Repository. 🌟

You are free to [Follow Me](https://github.com/kaal-coder) !

**Comment by AdityaDhavale** [comment]
@kaal-coder I have added my self made project in Frontend Projects folder . Please merge my PR.

**Comment by kaal-coder** [comment]
> @kaal-coder I have added my self made project in Frontend Projects folder . Please merge my PR.

Merged.
You can checkout :- [hacktoberfest-for-all](github.com/kaal-coder/hacktoberfest-for-all)
follow if you wish :- [kaal-coder](https://github.com/kaal-coder)

**Comment by AdityaDhavale** [comment]
Hey @kaal-coder  the link [hacktoberfest-for-all](https://github.com/kaal-coder/hacktoberfest2023/issues/github.com/kaal-coder/hacktoberfest-for-all) shows page not found

**Comment by kaal-coder** [comment]
> Hey @kaal-coder the link [hacktoberfest-for-all](https://github.com/kaal-coder/hacktoberfest2023/issues/github.com/kaal-coder/hacktoberfest-for-all) shows page not found

please recheck

**Comment by Khushi-74** [comment]
@kaal-coder Hi, I have added my self made python project virtual desktop assistant please check my PR

**Comment by AdityaDhavale** [comment]
Hey @kaal-coder will u please assign me n then merge so that it will reflect in my hacktoberfest contribution. Will u pls do it cuz  I've  no  contributions so far

**Comment by Khushi-74** [comment]
hi @kaal-coder ,can you please assign me and then merge ,its not reflecting in my contributions

**Comment by kaal-coder** [comment]
@Khushi-74 is it showing now ?

**Comment by AdityaDhavale** [comment]
@kaal-coder   No

**Comment by Khushi-74** [comment]
> @Khushi-74 @AdityaDhavale is it showing now ?

No , on hacktoberfest page , the contribution to this repo has been marked excluded

**Comment by AdityaDhavale** [comment]
@kaal-coder Yeah on hacktoberfest page this page is marked excluded. Could u please look into it

**Comment by aayushi-0407** [comment]
@kaal-coder i have a self made project ml based can i give a pull request for same as my contribution for hacktoberfest?

**Comment by Kirti-Pant** [comment]
@kaal-coder  I've my self made machine learning project can I make a pull request for hacktoberfest2023 ??

**Comment by imdeveshshukla** [comment]
Please Assign me this issue

**Comment by kaal-coder** [comment]
@imdeveshshukla no need to get assigned. Directly raise a PR

**Comment by Vaibhav832** [comment]
#320 
Added

**Comment by aayushi-0407** [comment]
@kaal-coder I've added files in ML Project kindly merge

**Comment by lucashchang** [comment]
hey @kaal-coder I've added my project into Frontend Projects, please kindly merge, thank you.

**Comment by sohamhaldar** [comment]
hey @kaal-coder I have added my project in frontend projects . i have created a pull request of no #362 . please kindly merge it.
Thank You

**Comment by Kirti-Pant** [comment]
@kaal-coder I've made a PR please do have a look into my ML project

**Comment by Kirti-Pant** [comment]
@kaal-coder  please merge #388

**Comment by Apoorva57** [comment]
hey @kaal-coder i have added an ML Project and made pr for the same #389 , please do have a look

**Comment by Apoorva57** [comment]
hey @kaal-coder i have added a fully functional matrix transpose calculator in frontend project and a ui/ux design for NGO portal website, the PR for these are #412 and #410 respectively, please do have a look

**Comment by Diptanghsu-DC** [comment]
@kaal-coder i have added 2 projects of python and ML . Please merge them.
